static_frame/core/series.py

```diff
@@ -187,6 +187,10 @@ def from_concat(cls,
             if index is None:
                 array_index.append(c.index.values)
 
+        # End quickly if empty iterable
+        if len(array_values) == 0:
+            return cls([], index=index, name=name)
```

Comment:
As below, can we initialize with something other than a list, such as EMPTY_TUPLE?